rero_ils/modules/ill_requests/models.py

```diff
@@ -33,3 +33,12 @@ class ILLRequestIdentifier(RecordIdentifier):
         db.BigInteger().with_variant(db.Integer, 'sqlite'),
         primary_key=True, autoincrement=True,
     )
+
+
+class ILLRequestStatus:
```

Comment:
cosmetic: could be inherited of object : `class ILLRequestStatus(object):`

Reply:
In Python 3, you don't need to inherit from object. If you don't specify a parent class, you inherit from `object` by default.